modules/backend/behaviors/FormController.php

```diff
@@ -177,7 +177,7 @@ public function initForm($model, $context = null)
          * Detected Relation controller behavior
          */
         if ($this->controller->isClassExtendedWith(\Backend\Behaviors\RelationController::class)) {
-            $this->controller->initRelation($model);
+            $this->controller->initRelation(clone $model);
```

Comment:
Just to explore the possibility, could we always operate with a clone in the `RelationController::initRelation()` method? Then this line wouldn't need to change, and that way there's no subtle behavior change when `RelationController` is being used with `FormController` vs. when it's being used on its own.

Reply:
My personal preference would be to keep it as it is, if someone wants similar isolation they can initialize it the same way on their own, but if we do the clone in the relationcontroller it makes it impossible for them to choose to have the old behaviour.